### PR TITLE
Feature conversation

### PR DESCRIPTION
**Pull request by fedescarpa**
bitmoji

Fixes https://github.com/mumuki/mumuki-platform/issues/169
Fixes https://github.com/mumuki/mumuki-platform/issues/196
Related to https://github.com/mumuki/mumuki-platform/issues/235

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ gulp.task('release', (done) => {
 
 gulp.task('config', () => {
   return gulp.src(`${configFile()}`)
+    .pipe(replaceEnvVar('MUMUKI_LABORATORY_URL'))
```
Comment (flbulgarelli) [inline gulpfile.js]: Two times?

File: src/scripts/config/strings/lang_es.js [hunk 2]
```diff
@@ -89,16 +99,20 @@ angular
...
+    new_message: 'Nuevo mensaje',
+    new_student: 'Nuevo estudiante',
+    new_teacher: 'Nuevo docente',
     next_exercise_description: 'Ir al ejercicio siguiente',
     next_solution_description: 'Ir a la solución siguiente',
     no: 'No',
     no_courses_to_show: 'No hay cursos para mostrar',
     no_exams_to_show: 'No hay exámenes para mostrar',
     no_guides_to_show: 'No hay guías para mostrar',
+    no_notifications_to_show: 'No hay notificaciones para mostrar',
```
Comment (aguspina) [inline src/scripts/config/strings/lang_es.js]: empty_notifications is more accurate

File: src/scripts/config/strings/lang_es.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ angular
     available_courses: 'Cursos disponibles',
     bulk: 'Masivo',
     by_last_submission_date: 'Por fecha de ultima solución',
+    by_messages: 'Por cantidad de mensajes',
```
Comment (fedescarpa) [inline src/scripts/config/strings/lang_es.js]: Done

File: src/scripts/config/strings/lang_es.js (hunk 2 above)
Comment (fedescarpa) [inline src/scripts/config/strings/lang_es.js]: Done

**Pull request merged**